info.yml

```diff
@@ -18,6 +18,30 @@ configuration:
                      commands that do not match any of the values in the groups
                      setting."
 
+    run_at_startup:
+        type: list
+        description: "Controls what apps will run on startup.  This is a list
+                     where each element is a dictionary with two keys:
+                     'app_instance' and 'name'.
+                     The 'app_instance' value connects this entry to a
+                     particular app instance defined in the environment
+                     configuration file.
+                     The 'name' is the menu name of the command to run when
+                     desktop starts up. If 'name' is '', then all commands from
+                     the given app instance are started.
+
+                     To add the 'Apps' tab (which contains the app launcher),
+                     add the following startup command to the list:
```

Comment:
Nope, that would make sense (I'm all for flexibility).  But in that case there should be a default value for this that does the right thing.  Just in case somebody ends up using add_engine to add tk-desktop to an environment.